plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -159,14 +159,6 @@ public static long resolveSnapshotId(Table table, long snapshotId)
         return columns.build();
     }
 
-    public static String getDataPath(String location)
```

Comment:
doesn't seem to belong to "pass in table properties to initialize location provider in sink prov…" commit